values.yaml

```diff
@@ -218,6 +218,27 @@ postgres:
   # the PostgreSQL connection. For acceptable values, see:
   #  https://www.postgresql.org/docs/9.1/libpq-ssl.html
```

Comment:
maybe want to link to the version of the min version of pg we support: https://www.postgresql.org/docs/11/libpq-ssl.html